src/Search/Workflow/ResearchArticleWorkflow.php

```diff
@@ -132,7 +132,7 @@ public function index(ArticleVersion $article) : array
             'value' => json_encode($articleObject->dataSets ?? '[]'),
         ];
 
-        $sortDate = $article->getStatusDate() ? $article->getStatusDate() : $article->getPublishedDate();
+        $sortDate = $article->getStatusDate();
```

Comment:
Makes sense to remove. If there's no status date, there won't be a published date either.